### DISCUSSION TITLE
jira issue edit --status

### DISCUSSION THREAD

**Discussion by realtime-neil** [ankitpokhrel/jira-cli#212] [Ideas] [answered]
**Is your feature request related to a problem? Please describe.**
No. 

**Describe the solution you'd like**
I would like the ability to edit the status of an issue; e.g.,
```
jira issue edit FOOBAR-1024 --status "Closed"
```

**Describe alternatives you've considered**
I have considered the atlassian web page. The atlassian web page sucks.

**Additional context**
Not applicable.

**Comment by realtime-neil** [comment]
@ankitpokhrel how do I move this to the "ideas" category?

**Reply by ankitpokhrel** [reply]
Moved. You can use [Discussions page ](https://github.com/ankitpokhrel/jira-cli/discussions)to start a discussion in `ideas` category.

**Comment by ankitpokhrel** [comment, marked as answer]
@realtime-neil how is this different from [move/transition](https://github.com/ankitpokhrel/jira-cli#movetransition) feature? Or, did I miss something 🤔

```sh
$> jira issue move -h
Move transitions an issue from one state to another.

USAGE
  jira issue move ISSUE-KEY STATE [flags]

FLAGS
  -h, --help   help for move
      --web    Open issue in web browser after successful transition

INHERITED FLAGS
  -c, --config string    Config file (default is /Users/ankitpokhrel/.config/.jira/.config.yml)
      --debug            Turn on debug output
  -p, --project string   Jira project to look into (defaults to /Users/ankitpokhrel/.config/.jira/.config.yml)

ARGUMENTS
  ISSUE-KEY     Issue key, eg: ISSUE-1
  STATE         State you want to transition the issue to

EXAMPLES
  $ jira issue move ISSUE-1 "In Progress"
  $ jira issue move ISSUE-1 Done

ALIASES
  transition
  mv

LEARN MORE
  Use 'jira <command> <subcommand> --help' for more information about a command.
```

**Reply by realtime-neil** [reply]
@ankitpokhrel whoops, I think you might be right about that; `jira issue move` is what I wanted. Thank you and please disregard this issue.
